Css Customization (#215)
jellyfin/jellyfin-docs: general/clients/skins.md

---
uid: css-custom
title: CSS Customization
---

# CSS Customization

Custom CSS provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. Currently, the css modifications only work on WEBUI. To implement these changes, go to Dashboard > General > Custom CSS.

![image](https://user-images.githubusercontent.com/20715731/73392971-d1cc7d80-42a8-11ea-8552-3d311655ea37.png)

## Color Modifications

> ### Colors Codes:<br>

Reviewer: Not sure header elements need a break.

---
uid: css-custom
title: CSS Customization
---

# CSS Customization

> To provide customization within Jellyfin such as changing colors or modifying behavior. Here are a collection of community scripts available. While the css scripts should work between all clients, there may be differences in behavior between the different clients. To implement these changes, go to Dashboard > General > Custom CSS.

Reviewer: ```suggestion
Custom CSS provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. While the css scripts should work between all clients, there may be differences in behavior between the different clients. To implement these changes, go to Dashboard > General > Custom CSS.
```

Author: done

![image](https://user-images.githubusercontent.com/20715731/73392971-d1cc7d80-42a8-11ea-8552-3d311655ea37.png)

## Color Modifications

### Colors Codes:<br>
Yellow: `#d0d000`<br>
Green: `#5dd000`<br>
Black: `#000000`<br>
Blue: `#0000d0`<br>

> Red: `#d00000`<br>

Reviewer: maybe just link to a hex color picker instead

Author: gave just the primary colors and a link to a hex chart.

> 

Reviewer: only one newline needed

Author: done

> #### Change Played Icon (Web Only): 

Reviewer: I would just say that all CSS customizations will only work on the web client for now.

Author: Updated the intro paragraph to reflect that and removed the web only tag.

---

> uid: css-custom

Reviewer: ```suggestion
uid: css-customization
```

title: CSS Customization
---

> # CSS Customization

Reviewer: You'll need to add this page to the table of contents as well.

Author: done. added books as well.

> Custom CSS provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. Currently, the css modifications only work on WEBUI. To implement these changes, go to Dashboard > General > Custom CSS.

Reviewer: ```suggestion
Custom CSS provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. Currently, the CSS modifications only work on the web client. To implement these changes, go to Dashboard > General > Custom CSS.
```

Author: This will need to be reworded once https://github.com/jellyfin/jellyfin-web/pull/763 is flushed out.

![image](https://user-images.githubusercontent.com/20715731/73392971-d1cc7d80-42a8-11ea-8552-3d311655ea37.png)

## Color Modifications

### Colors Codes:

> Here are a few examples, see [here](https://htmlcolorcodes.com/color-picker/) for a hex color chart.<br>

Reviewer: ```suggestion
Here are a few examples, see [here](https://htmlcolorcodes.com/color-picker/) for a hex color chart.

```

Green: `#5dd000`<br>
Blue: `#0000d0`<br>
Red: `#d00000`<br>

#### Played Indicator: 

`.playedIndicator { background: #5dd000 !important; }`

> Before:<br>

Reviewer: ```suggestion
Before:

```

![image](https://user-images.githubusercontent.com/20715731/73392328-97aeac00-42a7-11ea-817f-7234b8a78783.png)

> Green Mod:<br>

Reviewer: ```suggestion
Green Mod:

```

![image](https://user-images.githubusercontent.com/20715731/73392302-86659f80-42a7-11ea-9a9a-222cbbe466c6.png)

#### Background Color:

`.backgroundContainer, .dialog, html { background-color: #0fd0d0 !important; }`

> #### Hotdogs with some catsup (let's not use these colors lol)

Reviewer: ```suggestion
#### Hotdogs and Catsup:
```

```
.skinHeader, .mainDrawer, .emby-input, .emby-textarea, .emby-select, .navMenuOption-selected, .cardBox, .paperList {
	background: #ff9475;
}
.emby-input, .emby-textarea, .emby-select {
	border-color:#fdbe7d;
}

.backgroundContainer.withBackdrop, .backdropContainer, .backgroundContainer{
	background: #fdbe7d;
}

#myPreferencesMenuPage .listItemBodyText,
.emby-tab-button[data-index="0"],
#myPreferencesMenuPage > div > div > div > a:nth-child(odd),
.button-submit,
.mainAnimatedPage *:nth-child(odd),
.dashboardGeneralForm *:nth-child(odd),
.mainDrawer-scrollContainer *:nth-child(odd),
.headerRight *:nth-child(odd) {
	color: red;
}

#myPreferencesMenuPage .listItemIcon,
.emby-tab-button[data-index="1"],
#myPreferencesMenuPage > div > div > div > a:nth-child(even),
.mainAnimatedPage *:nth-child(even),
.dashboardGeneralForm *:nth-child(even),
.mainDrawer-scrollContainer *:nth-child(even),
.headerRight *:nth-child(even)
.cancel {
	color: yellow;
}
```

> 

Reviewer: too many newlines
